bootstrap-joining-demo/marathon-api/src/main/resources/application.conf

```diff
@@ -0,0 +1,39 @@
+akka {
+
+  discovery.method = aws-api-ec2-tag-based
+
+  actor {
+    provider = "cluster"
+  }
+
+  remote {
+    netty.tcp {
+      port = 2551
```

Comment:
@longshorej Looks like these port settings can be omitted here, because they get overridden in the start command defined in `app.json`